### ISSUE TITLE
how to finetune ?

### ISSUE THREAD

**Issue by dreamer121121** [SHI-Labs/FcF-Inpainting#27]
I use --resume places.pkl to finetune on my own datasets, but the result is not good, even worse than places.pkl,,,,i want to know if should i change the lr or other params?

**Comment by praeclarumjj3** [comment]
Hi @dreamer121121, thanks for your interest.

Please provide more information. It could be that your dataset's domain is completely different than the places2 domain, in which case fine-tuning might not work very well. Usually, on new datasets, you must carefully tune the lr/batch size depending on the dataset size to obtain optimal performance. You may also need to tune the loss weights.
